index.py

```diff
@@ -67,7 +70,7 @@ class VSM:
     """
     def __init__(self, in_dir, d_file, p_file):
         self.dictionary = {}  # content, title, court
-        self.docid_set = set() # docID
+        self.docid_set = {} # (doc_id:top K most common terms for that doc_id) mappings
```

Comment:
Nit: Change name to, cos it isnt a set anymore

Reply:
renamed to docid_term_mappings